webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'development',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  module: {
```

Comment:
✅ Webpack is used to set up the project 👍🏻 
✅ It has an HTML file configured in the Webpack config settings as a template using the *HTML Webpack Plugin* 👍🏻